app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx

```diff
@@ -37,13 +49,39 @@ const Form = ({ onClose, refetch, note, isEdit }) => {
               className="w-full flex-grow-0"
               label="Title"
               name="title"
+              placeholder="Enter note title"
+              unlimitedChars={false}
             />
             <Textarea
               required
               className="w-full flex-grow-0"
               label="Description"
               name="description"
-              rows={8}
+              placeholder="Enter note description"
+              rows={1}
+              size="small"
+              unlimitedChars={false}
+            />
+            <Select
+              isSearchable
+              required
+              className="w-full flex-grow-0"
+              label="Assigned Contact"
+              name="assignedContact"
+              optionRemapping={{}}
+              options={DUMMY_CONTACTS}
+              placeholder="Select Role"
```

Comment:
Is it supposed to be role or contact? Please be consistent with your naming.

Reply:
Hey,
The given figma design contains similar inconsistency.
<img width="396" alt="image" src="https://github.com/sritabh/neeto-ui-challenge-by-sritabh/assets/43126956/d0c94a25-56ed-4c53-ad6c-dac7d21daa06">
Hence the confusion, will resolve it.

Reply:
Got it. I will ask the designers to make the changes.